must-gather/node-gather/daemonset.yaml

```diff
@@ -64,3 +66,7 @@ spec:
         hostPath:
           path: /var/lib/kubelet/pod-resources
           type: Directory
+      - name: log
```

Comment:
ok, this is needed because of this: https://github.com/openshift-kni/performance-addon-operators/blob/master/tools/gather-sysinfo/gather-sysinfo.go#L78

So when we create the archive, the code looks for `root` + path, so in `/host/var/log/machineinfo.json`.

So adding this mount works, but it's unnecessary and actually wrong: we should not write back in the host filesystem.

So we need to change a bit how we declare the paths to add in the archive.